products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Amazon RDS for MySQL
+category: db
```

Comment:
```suggestion
category: service
```

RDS is not a installable database, it is a cloud-service (similar to how we track EKS as service, and Kubernetes as `server-app`).